internal/kms/vault.go

```diff
@@ -422,7 +422,7 @@ func (kms *vaultKMS) StoreDEK(key, value string) error {
 	}
 
 	pathKey := filepath.Join(kms.vaultPassphrasePath, key)
-	err := kms.secrets.PutSecret(pathKey, data, kms.keyContext)
+	_, err := kms.secrets.PutSecret(pathKey, data, kms.keyContext)
```

Comment:
add a comment on what exactly is getting discarded here. same in other places as well.